docs/docs/how-to/start-developing.md

```diff
@@ -0,0 +1,167 @@
+
+```
+pacman -Sy --needed --noconfirm base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja mingw-w64-x86_64-clang mingw-w64-x86_64-lld wget
+```
+
+
+Next, we will set the Mingw-w64 path to the environment variables. This will allow the terminal to search the Mingw-w64 directory for executables to run without requiring the user to specify the full address of the .exe each time.
+
+Search for “Edit environment variables for your account.” in the Window’s search bar and select the option that appears.
+
+!!! help
+    You can also access the Environment Variables window from "Control Panel → User Accounts → User Accounts → Change my environment variables".
+
+Double-click on “Path” under “User variables”.
+
+![Double-click on the “Path” user variable.](assets/set-environment-variable-mingw64-path.png)
+
+In the “Edit environment variable” window, select “New” and add `C:\msys64\mingw64\bin` to the empty field that appears.
+```
+C:\msys64\mingw64\bin
+```
+
+![Add the mingw64 bin directory to the environment variables.](assets/set-environment-variable-mingw64-new.png)
+
+## Rust ##
+
+Rust is a programming language that we will be using. Install it [here](https://www.rust-lang.org/tools/install). 
+
+Open up the Command Prompt. You can use the Windows Search Bar to find the Command Prompt application.
+
+![Search for the Command Prompt in the Window’s search bar.](assets/search-command-prompt.png)
+
+!!! tip
+    You can check if Rust is properly installed by inputting `cargo –version` in the Command Prompt. 
+
+    If a version number is printed, your installation was successful!
+        ```
+        cargo –version
+        ```
+
+Input `rustup default nightly` to use the nightly version of Rust. The nightly release is updated more frequently compared to the stable and beta versions.
+	```
+    rustup default nightly
+	```
+
+Input the command `rustup target add wasm32-unknown-unknown --toolchain nightly` to add support for building to web browsers.
+	```
+    rustup target add wasm32-unknown-unknown --toolchain nightly
+	```
+
+Use the `rustup update` command to update your installation. Since we are using the nightly release, you are encouraged to update often.
+	```
+    rustup update
+	```
+ 
+
+##  Git & GitHub ##
+
+Git is a system used for managing and tracking changes in your code. It is also known as a Version Control System (VCS). It makes it easier for collaborators to work together, and allows you to access our projects too!
+
+Install Git [here](https://git-scm.com/download). The installation settings can be left unchanged.
+
+Although Git is powerful, it can be daunting to use on its own. Instead, we use GitHub, a service which employs Git’s version control with its own to make project collaboration much easier. 
+
+Download GitHub for Desktop [here](https://desktop.github.com/). You can install it with the default settings, and create a GitHub account when prompted if you do not have one.
```

Comment:
Would this be better?

```
Although Git is powerful, it can be daunting to use on its own. Instead, we use GitHub, a service which employs Git’s version control with its own to make project collaboration much easier. 

We recommend that you use a [Git GUI Client](https://git-scm.com/downloads/guis) to facilitate your work by pruning off the manual input of code and making the version control process more visual.

We will be using [GitHub Desktop](https://desktop.github.com/) for the tutorials. You can install it with the default settings, and create a GitHub account when prompted if you do not have one.

!!! note
    Even though we recommend GitHub Desktop, feel free to use any GUI client of your choice!
```

Reply:
Seems fair enough. I didn't know there was a page with a list of GUIs.